GPSTest/src/main/java/com/chesapeaketechnology/gnssmonkey/AGpsMonkeyActivity.java

```diff
@@ -119,6 +124,24 @@ public void onRequestPermissionsResult(int requestCode, String[] permissions, in
         }
     }
 
+    /**
+     * Called immediately before the activity is made visible. This method should initialize
+     * components that are released in {@link #onStop()}.
+     *
+     * {@inheritDoc}
+     */
+    @Override
+    protected void onStart() {
+        // TODO KMB: If we are unbinding in stop, shouldn't we bind in start?
```

Comment:
Based on the Activity life cycle paths, I would agree.  It looks like it is possible for onStop to get called, but then not have onCreate to get called again, which would leave the service unbinded and not started.